src/sqs/push.ts

```diff
@@ -20,7 +18,7 @@ export type PushQueueMessagePayload = {
 	jiraHost: string,
 	installationId: number,
 	webhookId: string,
-	webhookReceived?: Date,
+	webhookReceived?: number,
```

Comment:
Some typing here was wrong, and surprisingly Typescript didn't catch it

Reply:
I had wondered about that when I implemented the other queues, but was afraid to touch it 👍🏼